watertap/unit_models/gac.py

```diff
@@ -248,7 +249,7 @@ def eq_isobaric_adsorbate(b, t):
         self.saturation_mtz_upstream = Param(
             default=0.95,
             initialize=0.95,
-            within={0.95, 0.99},
+            domain={0.95, 0.99},
```

Comment:
I guess you can do this, but I think we'd typically use `bounds=(0.95, 0.99)` in this case.

Reply:
This is a convention with defining the mass transfer zone upstream edge as typically 0.95 or 0.99 exclusively, but I just looked in literature and there are rare cases of 0.98, so I think converting to bounds is valid.

Reply:
@adam-a-a Actually looks like since Param() doesn't have a bounds argument, the solution would have to be 
```suggestion
            domain=NonNegativeReals,
```
with the doc string just stating the conventional values

Reply:
Ah, I overlooked the fact that this was a Param. In that case, no need to change to `NonNegativeReals` domain.